src/javascript/app/pages/trade/tradepage.js

```diff
@@ -28,7 +28,7 @@ const TradePage = (() => {
             el_iframe.src = `${iframe_target_origin}/localstorage-sync.html`;
         }
 
-        BinarySocket.wait('authorize').then(() => {
+        BinarySocket.wait('authorize', 'landing_company').then(() => {
```

Comment:
What is the reason behind adding `landing_company` here?

Reply:
because we need to get the landing_company shortcode to filter the markets and since it's async function we get undefined from State.getResponse('landing_company.gaming_company.shortcode')